### PR TITLE
Python plotly funnel

### LINKED ISSUE #5172: [Term Entry] Python Plotly - express .funnel()
### Reason/inspiration (optional)

We would like a new term entry on the `.funnel()` method of the [`express module`](https://www.codecademy.com/resources/docs/plotly/express) in Plotly, Python. The entry should go in a new file under `docs/content/plotly/concepts/express/terms/funnel/funnel.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by ebikatsudon**
### Description

Created term entry for the `.funnel()` method in Python Plotly Express module. Also added images docs\media\plotly-express-funnel-example1.png and docs\media\plotly-express-funnel-example2.png

### Issue Solved

Closes #5172 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/plotly/concepts/express/terms/funnel/funnel.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  - 'Methods'
+  - 'Plotly'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+The **.funnel()** method in Plotly Express creates a chart showing the progressive reduction, or "funneling," of data as it moves through sequential stages. The chart is composed of stacked horizontal bars, with the length of each bar representing a value at each stage in the process. This helps visualize how the values change as they move through each stage and can highlight bottlenecks or drop-offs in values.
```
Comment (SaviDahegaonkar) [inline content/plotly/concepts/express/terms/funnel/funnel.md:20]:
```suggestion
The **`.funnel()`** method in Plotly Express creates a chart showing the progressive reduction, or "funneling," of data as it moves through sequential stages. The chart is composed of stacked horizontal bars, with the length of each bar representing a value at each stage in the process. This helps visualize how the values change as they move through each stage and can highlight bottlenecks or drop-offs in values.
```
The method name should be included inside (`.....`) single backticks.

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @ebikatsudon ,
The entry LGTM, I have suggested a change please make it asap.

Thanks,
Savi

**Comment by ebikatsudon** [comment]
Hi @SaviDahegaonkar, thank you for your review! I have made and committed the change you suggested.

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @ebikatsudon ,
LGTM! This entry looks good for second round of review.

Thanks,
Savi

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing @ebikatsudon 😄 

The entry is ready to be merged! 🚀

**Pull request merged**